<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1024</width>
                <height>768</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Multi-Robot Motion Planning</string>
        </property>
        <widget class="QScrollArea" name="centralwidget">
            <layout class="QGridLayout">
                <item row="0" column="0">
                    <widget class="QGraphicsView" name="graphicsView">
                        <property name="focusPolicy">
                            <enum>Qt::StrongFocus</enum>
                        </property>
                        <property name="verticalScrollBarPolicy">
                            <enum>Qt::ScrollBarAlwaysOn</enum>
                        </property>
                        <property name="horizontalScrollBarPolicy">
                            <enum>Qt::ScrollBarAlwaysOn</enum>
                        </property>
                        <property name="transformationAnchor">
                            <enum>QGraphicsView::NoAnchor</enum>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar" />
        <widget class="QToolBar" name="toolbar">
            <property name="windowTitle">
                <string>Visualization tools</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <!-- Add actions for the main toolbar -->
            <addaction name="actionInsertNone" />
            <addaction name="separator" />
            <addaction name="actionInsertWorkspace" />
            <addaction name="separator" />
            <addaction name="actionInsertStartConfigs" />
            <addaction name="separator" />
            <addaction name="actionInsertTargetConfigs" />
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1024</width>
                    <height>32</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionNew" />
                <addaction name="actionOpen" />
                <addaction name="separator" />
                <addaction name="actionImport" />
                <addaction name="actionExport" />
                <addaction name="separator" />
                <addaction name="actionExit" />
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>Edit</string>
                </property>
                <addaction name="actionUndo" />
                <addaction name="actionRedo" />
                <addaction name="separator" />
                <addaction name="actionInsertNone" />
                <addaction name="actionInsertWorkspace" />
                <addaction name="actionInsertStartConfigs" />
                <addaction name="actionInsertTargetConfigs" />
            </widget>
            <widget class="QMenu" name="menuTools">
                <property name="title">
                    <string>Tools</string>
                </property>
                <addaction name="actionGenerateFreeSpace" />
                <addaction name="actionGenerateMotionGraph" />
            </widget>
            <widget class="QMenu" name="menuView">
                <property name="title">
                    <string>View</string>
                </property>
                <addaction name="actionRecenter" />
            </widget>

            <addaction name="menuFile" />
            <addaction name="menuEdit" />
            <addaction name="menuTools" />
            <addaction name="menuView" />
        </widget>

        <!-- Actions -->
        <action name="actionNew">
            <property name="text">
                <string>&amp;New</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionOpen">
            <property name="text">
                <string>&amp;Open</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionImport">
            <property name="text">
                <string>&amp;Import</string>
            </property>
        </action>
        <action name="actionExport">
            <property name="text">
                <string>&amp;Export</string>
            </property>
        </action>
        <action name="actionExit">
            <property name="text">
                <string>E&amp;xit</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Q</string>
            </property>
        </action>

        <action name="actionUndo">
            <property name="text">
                <string>&amp;Undo</string>
            </property>
            <property name="shortcut">
                <string>Qtrl+Z</string>
            </property>
        </action>
        <action name="actionRedo">
            <property name="text">
                <string>&amp;Redo</string>
            </property>
            <property name="shortcut">
                <string>Qtrl+Y</string>
            </property>
            <property name="shortcut">
                <string>Qtrl+Shift+Z</string>
            </property>
        </action>
        <action name="actionInsertNone">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>&amp;Disable insertion mode</string>
            </property>
        </action>
        <action name="actionInsertWorkspace">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>&amp;Workspace insertion mode</string>
            </property>
        </action>
        <action name="actionInsertStartConfigs">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>&amp;Start configuration insertion mode</string>
            </property>
        </action>
        <action name="actionInsertTargetConfigs">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>&amp;Target configuration insertion mode</string>
            </property>
        </action>

        <action name="actionGenerateFreeSpace">
            <property name="text">
                <string>Generate &amp;Free-space</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+F</string>
            </property>
        </action>
        <action name="actionGenerateMotionGraph">
            <property name="text">
                <string>Generate &amp;Motion graph</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+M</string>
            </property>
        </action>

        <action name="actionRecenter">
            <property name="text">
                <string>Re&amp;center the viewport</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+R</string>
            </property>
        </action>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
